apps/backend/src/middlewares/checkFunctions/electionCheck.ts

```diff
@@ -0,0 +1,36 @@
+import type { Election, User } from '@repo/votura-validators';
+import { db } from '../../db/database.js';
+
+export async function electionExists(electionId: Election['id']): Promise<boolean> {
```

Comment:
IMHO: Would rename this to `existsElection` because it's much more clear to me what type will be returned.